docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -18,6 +18,7 @@ Template: # Power name        # Must be unique
     Range: X                  # Integer of spaces. 0 for self, 1 for melee
     AOE: Shape X              # Aura, cone, or line followed by integer for size
     Target: X                 # Integer, # of targets
+    Duration: X               # String, to specify '1 hour' or '10 minutes' or '1 round'
```

Comment:
Do you expect this to appear as part of the merged printed mechanic? 

In some cases (e.g., create poison) duration could refer to the (a) time it takes to manifest the product (e.g., 10m crafting time), (b) the available time for the product to be used, or (c) the duration of the effects once used. We might need to adjust the structure of the yaml spec to differentiate the power's casting time, availability once cast, and duration of the effects in place. Currently, we use power type (major, minor) to communicate casting time, but this is a vestige of a previous XP-based system

Reply:
I think it will be an important key to store in the YAML but not sure how I'd want it merged for printing. Duration should be for the length of the effect, and to me casting time is either a Minor Action, a Major Action, or specified in the Mechanic. Available time of product (i.e. when created poison disappears) is typically written out in the Mechanic specifically and I don't think it will be prevalent enough to add in as a YAML key compared to Duration of the effect.

Reply:
After discussion, we resolved to add a `Channel` power type, the mechanic of which will specify a use-time duration other than 'Major' or 'Minor' action